src/view/form-edit-view.js

```diff
@@ -310,9 +335,21 @@ export default class FormEditView extends SmartView {
   }
 
   #cityToggleHandler = (evt) => {
-    const newCity = evt.target.value;
+    let newCity = '';
+
+    for(let i = 0; i < CITIES.length; i++) {
+      if(CITIES[i] === evt.target.value) {
```

Comment:
```suggestion
      if (CITIES[i] === evt.target.value) {
```

Точно пробел устал)))